imap_processing/cdf/config/imap_idex_l1a_variable_attrs.yaml

```diff
@@ -1,35 +1,46 @@
 int_fillval: &int_fillval -9223372036854775808
-min_epoch: &min_epoch -315575942816000000
-max_epoch: &max_epoch 3155630469184000000
+
+max_uint32: &max_uint32 4294967295
+max_uint16: &max_uint16 65535
+max_uint8: &max_uint8 255
 
 data_min: &data_min 0
 data_max: &data_max 4096
 
+string_base_attrs: &string_base
+  CATDESC: string metadata
+  FIELDNAM: string_metadata
```

Comment:
I wonder if these should be removed so that `CATDESC` and `FIELDNAM` are left undefined if the the attribute using `string_base_attrs` doesn't explicitly define them? I am just thinking of a case where we accidentally leave in `string_metadata` as the value of `CATDESC` in the CDF, which would be strange to see in the final data product.

Alternatively we could use a value of `TBD` or `Undefined` so it is a bit more clear that the value doesn't mean anything.

Reply:
Good idea. I can leave them blank